src/android/logging/src/main.rs

```diff
@@ -25,5 +25,6 @@ fn main() {
             .with_min_level(log::Level::Trace),
     );
     debug!("Starting program.");
+    info!("Things are going fine.");
```

Comment:
I believe you need to import `info!` first.

Reply:
Indeed! Done.

Reply:
Cool :smile: I need to hook up the `build_all.sh` script to CI, but it seems that it'll be a little tricky to do an AOSP checkout in a reasonable time.